build.ps1

```diff
@@ -256,12 +256,23 @@ function cleanExtension([string] $bitness) {
     Remove-Item -Recurse -Force "$privateSiteExtensionPath\$bitness\runtimes\linux" -ErrorAction SilentlyContinue
     Remove-Item -Recurse -Force "$privateSiteExtensionPath\$bitness\runtimes\osx" -ErrorAction SilentlyContinue
 
-    Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\node\grpc\src\node\extension_binary" | 
+    Get-ChildItem "$privateSiteExtensionPath\$bitness\workers\node\grpc\src\node\extension_binary" -ErrorAction SilentlyContinue | 
     Foreach-Object {
         if (-Not ($_.FullName -Match "win32")) {
             Remove-Item -Recurse -Force $_.FullName
         }
     }
+
+    if ($bitness -eq "64bit") {
+        $keepRuntimes = @('win', 'win-x64', 'win10-x64')
+    } elseif ($bitness -eq "32bit") {
+        $keepRuntimes = @('win', 'win-x86', 'win10-x86')
```

Comment:
One caveat. The 32bit version of the Worker will not work on a 64bit OS because the `dotnet.exe` on the PATH that we rely on is the 64bit version.

If a 32bit OS is used, or the dotnet.exe on the path is the 32bit executable then the worker should work.

Is this going to be a problem?

@pragnagopa @fabiocav @alrod

Reply:
Have you verified this on the actual worker? The .NET Core SDK and runtimes installed there (therefore, what would be on the path) should all be 32-bit.

Reply:
yes, we are using self-contained approach for x64